[2021-12-30 11:41:11,239] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: cleansing_dag.cleanse_kym 2021-12-29T11:41:09.897762+00:00 [queued]>
[2021-12-30 11:41:11,255] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: cleansing_dag.cleanse_kym 2021-12-29T11:41:09.897762+00:00 [queued]>
[2021-12-30 11:41:11,256] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-12-30 11:41:11,257] {taskinstance.py:1095} INFO - Starting attempt 1 of 2
[2021-12-30 11:41:11,257] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-12-30 11:41:11,276] {taskinstance.py:1114} INFO - Executing <Task(PythonOperator): cleanse_kym> on 2021-12-29T11:41:09.897762+00:00
[2021-12-30 11:41:11,283] {standard_task_runner.py:52} INFO - Started process 14133 to run task
[2021-12-30 11:41:11,289] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cleansing_dag', 'cleanse_kym', '2021-12-29T11:41:09.897762+00:00', '--job-id', '37', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/kym_cleansing.py', '--cfg-path', '/tmp/tmps5r8l3kp', '--error-file', '/tmp/tmp1o11psg4']
[2021-12-30 11:41:11,290] {standard_task_runner.py:77} INFO - Job 37: Subtask cleanse_kym
[2021-12-30 11:41:11,343] {logging_mixin.py:109} INFO - Running <TaskInstance: cleansing_dag.cleanse_kym 2021-12-29T11:41:09.897762+00:00 [running]> on host 2eb70c976689
[2021-12-30 11:41:11,411] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cleansing_dag
AIRFLOW_CTX_TASK_ID=cleanse_kym
AIRFLOW_CTX_EXECUTION_DATE=2021-12-29T11:41:09.897762+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-29T11:41:09.897762+00:00
[2021-12-30 11:44:20,190] {python.py:151} INFO - Done. Returned value was: None
[2021-12-30 11:44:20,203] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=cleansing_dag, task_id=cleanse_kym, execution_date=20211229T114109, start_date=20211230T114111, end_date=20211230T114420
[2021-12-30 11:44:20,253] {local_task_job.py:151} INFO - Task exited with return code 0
